### PR TITLE
Fix typos on gh-pages

### PR DESCRIPTION
**Pull request by Aareon**
Replaced `suites` with `suits` in `suites my needs`. Added a direct link to the issues page in `open an issue` Replaced `hickups` with `hiccups`

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -236,8 +236,8 @@ <h2>is this ready for use yet?</h2>
 		<p>
       kinda! there are likely some major bugs in workspacer that I have not fixed yet, 
       but it is more or less functionally complete. i am using workspacer 24/7 at home and at 
-      work at the moment, so it suites my needs (aside from the occasional hickup or crash). 
-      if you run into any problems, please open an issue!
+      work at the moment, so it suits my needs (aside from the occasional hiccup or crash). 
+      if you run into any problems, please <a href="https://github.com/workspacer/workspacer">open an issue</a>!
```
Comment (josteink) [inline index.html:240]: Shouldnt this link also point to the issues sub-page? :)

**Comment by josteink** [comment]
Hey there and thanks for the corrections.

One small comment, and I'm sure we can merge this.

File: index.html (hunk 1 above)
Comment (Aareon) [reply index.html:240]: @josteink if we could create a default template for issues I think it would be beneficial.

**Comment by Aareon** [comment]
Updated `index.html` to link to the new issue creator. I think it would be awesome to set up a default issue template.

**Comment by josteink** [comment]
Awesome. Merged!

>  I think it would be awesome to set up a default issue template.

Have it your way. I've added you to the workspacer organization ;)

**Pull request merged**